angr/errors.py

```diff
@@ -396,3 +396,17 @@ class SimZeroDivisionException(SimException, SimOperationError):
 
 class AngrNoPluginError(AngrError):
     pass
+
+#
+# Concrete Targets Execution errors
+#
+
+class ConcreteMemoryError(AngrError):
```

Comment:
These three new errors are all raised inside engines. Please prepend them with `Sim` to follow the norm.